UmpleToJava/UmpleTLTemplates/JavaClassGenerator.ump

```diff
@@ -60,7 +60,7 @@ class JavaClassGenerator {
     JavaClassGenerator <<!<</*JavaClassGenerator*/>><<@ UmpleToJava.UmpleNotice >>
 <<#
   UmpleClass uClass = (UmpleClass) uElement;
-  globalUmpleClass = uClass;  
+  globalUmpleClass = uClass;
```

Comment:
You are just takng out spaces at the end of the line here. Almost all other changes in this file are similar to this. What is the point of the changes in this file. It just makes reviewing harder. It is OK to clean up spaces and indentation, but that should be done as a separate refactoring activity.